.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -0,0 +1,72 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+
+  # MacOS specific build settings
+  CIBW_BEFORE_ALL_MACOS: |
+    brew uninstall --force oclint
+    brew install gcc libomp
+
+  # Python build settings
+  CIBW_BEFORE_BUILD: |
+    pip install pybind11
+
+  # Testing of built wheels
+  CIBW_TEST_REQUIRES: numpy scipy pytest pytest-cov pytest-mock flaky
+
+  # Note: breaking up tests so that no test cases hang on Windows, to be reverted
```

Comment:
I didn't follow what this comment was referring to

Reply:
Nor I: I think this may be a left-over from a previous refactor. I think it will be safe to remove.

Reply:
Oh, I can explain this one. It was a comment added in this PR here: https://github.com/PennyLaneAI/pennylane-lightning/pull/59

In particular, the test `test_four_qubit_random_circuit` was hanging on Windows, likely due to an issue with pytest. We were never able to work out why it hanged, _but_ it must have been something to do with previous tests not cleaning up?

Because if we ran this test by itself, it didn't hang.